addons/account/views/account_move_views.xml

```diff
@@ -1336,7 +1336,7 @@
                                 </group>
                             </page>
                             <page id="aml_tab" string="Journal Items" groups="account.group_account_readonly" name="aml_tab">
-                                <field name="journal_line_ids"
+                                <field name="line_ids"
```

Comment:
Please be careful, this change is not stable
It broke many DB
https://github.com/odoo/odoo/pull/237274
https://github.com/odoo/enterprise/pull/100426

Reply:
Thanks for the heads up (and the fix) :+1: 
Sorry about that :blush: